stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -345,10 +367,11 @@ function externDecodePrivateKey(KeyStore|TrustStore keyStore, handle keyAlias, h
 } external;
 
 # Read a public key from the provided PKCS#12 archive file.
+# ```ballerina crypto:PublicKey|crypto:Error publicKey = crypto:decodePublicKey(keyStore, keyAlias); ```
 #
 # + keyStore - Key store or Trust store configurations
 # + keyAlias - Key alias
-# + return - Reference to the public key or `Error` if private key was unreadable
+# + return - Reference to the public key or `crypto:Error` if private key was unreadable
```

Comment:
```suggestion
# + return - Reference to the public key or `crypto:Error` if private key was unreadable
```
Please refer the template for the correct format.

Reply:
Corrected.